src/js/tabular_comparison/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import TabularComparison from "./tabular-comparison";
+import {createRoot} from "react-dom/client";
+
+//css
+
+//js
```

Comment:
If these comments aren't going to be in use they can be removed

Reply:
removed 👍